packages/core/src/components/accordion/components/accordion-section.tsx

```diff
@@ -218,6 +219,8 @@ export class AccordionSection implements ComponentInterface {
 
                 <span>{this.handleTitle}</span>
 
+                {isNeutral && <dso-info-button />}
```

Comment:
Is `<dso-info-button />` wel de meest ideale oplossing? Binnen dit component zit mogelijk een focusbaar element, en het bevat ook andere functionaliteit. In de neutrale variant willen we vooral het icoon als decoratief element. Misschien kunnen we die beter direct plaatsen met `<dso-icon>`

Reply:
Heb er `{isNeutral && <dso-icon class="info-icon" icon={this.open || this.hover ? "info-active" : "info"} />}` van gemaakt. de class `info-icon` is om de groene kleur te geven.

Reply:
:+1: